src/parse-options.js

```diff
@@ -13,6 +17,17 @@ const DEFAULT_OPTIONS = {
 function parseOptions (providedOptions) {
   let options = Object.assign({}, DEFAULT_OPTIONS, providedOptions)
 
+  // normalizing options.color to be a boolean or a color value
+  if (!(options.color in styles)) {
```

Comment:
Supplied or not, options.color will have a value. Is checking for a valid color not sufficient?